### DISCUSSION TITLE
Display publishing date for F1 TV content

### DISCUSSION THREAD

**Discussion by recursiveGecko** [f1multiviewer/issue-tracker#318] [Feature requests]
It would be nice to see publishing dates of individual videos and "last updated" dates for collections, assuming this data is available.

Professional design sketch, this one's free!

![image](https://user-images.githubusercontent.com/1461520/215193246-e72e65b3-2b7e-49d5-a824-aa54b946cb12.png)

Additionally, it would be handy to have a chronological "feed" of every video published on F1 TV.

**Comment by LapsTimeOFF** [comment]
I do understand why for the ”Playlists” but why for the videos ?

**Reply by LapsTimeOFF** [reply]
Actually nevermind, didn't saw that it was "Published" on the videos and not "Last Updated", my bad
